spec/lib/app_secrets_spec.rb

```diff
@@ -3,8 +3,8 @@
 RSpec.describe AppSecrets, type: :lib do
   let(:namespace) { "secret-namespace" }
   let(:key) { "SECRET_KEY" }
-  let(:secret_stub) { instance_double("Vault::Kv", data: { value: 1 }) }
-  let(:vault_stub) { instance_double("Vault::Kv", read: secret_stub) }
+  let(:secret_stub) { instance_double(Vault::Secret, data: { value: 1 }) }
```

Comment:
Vault.read returns `Vault::Secret`, not `Kv`
https://github.com/hashicorp/vault-ruby/blob/6f98a377218adff3bd180d6958426a130071306c/lib/vault/api/kv.rb#L56-L66

Reply:
No wonder! I couldn't get it last time I tried routine rubocop cleanup